hack/verify-govet.sh

```diff
@@ -18,7 +18,7 @@ cd "${OS_ROOT}"
 mkdir -p _output/govet
 
 FAILURE=false
```

Comment:
Line 8:
`if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.4') ]]; then`
don't forget to update that and `verify-golint.sh` and `verify-gofmt.sh`.